components/molecule/dataCounter/src/index.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import AtomButton from '@schibstedspain/sui-atom-button'
+import AtomInput, {inputSizes} from '@s-ui/react-atom-input'
+import MoleculeField from '@s-ui/react-molecule-field'
+
+import {withStateValue} from '@s-ui/hoc'
+
+const BASE_CLASS = `sui-MoleculeDataCounter`
+const CLASS_INPUT_CONTAINER = `${BASE_CLASS}-container`
+
+const MoleculeDataCounter = ({
+  id,
+  label,
+  value,
+  size = inputSizes.MEDIUM,
+  charsSize = 2,
+  placeholder = '1',
+  max = 99,
+  min = 1,
+  minValueHelpText = 'Minimum Value',
+  maxValueHelpText = 'Maximum Value',
+  onChange,
+  disabled
+}) => {
+  value = value || min
+
+  const isBelowMaxValue = value < max
+  const isOverMinValue = value > min
+  const isMaxValue = value === max
+  const isMinValue = value === min
+
+  const incrementValue = e => {
+    if (isBelowMaxValue) onChange(e, {value: value + 1})
+  }
+
+  const decrementValue = e => {
+    if (isOverMinValue) onChange(e, {value: value - 1})
+  }
+
+  const handleKeyDown = e => {
+    const {key} = e
+    if (key === 'ArrowUp') incrementValue(e)
+    if (key === 'ArrowDown') decrementValue(e)
+  }
+
+  let helpText
+  if (!disabled) {
+    if (isMaxValue && maxValueHelpText) helpText = maxValueHelpText
+    if (isMinValue && minValueHelpText) helpText = minValueHelpText
+  }
+
+  return (
+    <div className={BASE_CLASS}>
+      <MoleculeField name={id} label={label} helpText={helpText}>
+        <div
+          className={cx(
+            CLASS_INPUT_CONTAINER,
+            `${CLASS_INPUT_CONTAINER}--${size}`
+          )}
+        >
+          <AtomButton
+            disabled={disabled || value === min}
+            onClick={decrementValue}
+            type="secondary"
```

Comment:
you should use a constant for both button type props are it should be always the same